namara/__init__.py

```diff
@@ -28,3 +32,56 @@ def get_base_path(self, dataset, version):
     @staticmethod
     def is_aggregation(options):
         return options is not None and 'operation' in options.keys()
+
+    def get_dataset(self, orgId, projectId):
+        url= "https://api.namara.io/v0/organizations/"+orgId+"/projects/"+projectId+"/data_sets?offset=0&limit=250&api_key="+self.api_key
+        file = requests.get(url)
+        file = file.json()
+        num_datasets = len(file['data_sets'])
+
+        for i in range(num_datasets):
+            title = file['data_sets'][i]['data_set_metas'][0]['title'] #dataset name
+            ds_id = file['data_sets'][i]['id'] # gets dataset id
+            v_num = "en-"+str(len(file['data_sets'][i]['versions'])-1) # gets the latest version number
+            # creates the url for the lastset version of the dataset
+            ds_url = "https://api.namara.io/v0/data_sets/"+ds_id+"/data/"+v_num+"/export?geometry_format=wkt&api_key="+self.api_key+"&organization_id="+orgId+"&project_id="+projectId
```

Comment:
feels wrong to be hitting the export url in the api wrapper. Let me see what I can do